Project Doesn't Open in Android Studio on Mac

I was very excited to look into the code on this project, but when I tried to open it by importing it from GitHub into Android Studio, it didn't load. The reason I'm suspecting this is an issue with the repository and not with the IDE itself is because the error it gave me (see below) involved converting a URL from a local C:/ drive. I attached the log because I'm new to this and I may be completely wrong in my guess, but I'd love to be able to browse the code.
![screen shot 2015-11-17 at 3 07 49 pm](https://cloud.githubusercontent.com/assets/8755783/11223591/be278f82-8d3e-11e5-83a6-ccae16f93344.PNG)

```
Caused by: org.gradle.api.InvalidUserDataException: Cannot convert URL 'C:/' to a file.
	at org.gradle.api.internal.file.AbstractFileResolver.convertObjectToFile(AbstractFileResolver.java:188)
	at org.gradle.api.internal.file.BaseDirFileResolver.doResolve(BaseDirFileResolver.java:69)
	at org.gradle.api.internal.file.AbstractFileResolver.resolve(AbstractFileResolver.java:77)
	at org.gradle.api.internal.file.AbstractFileResolver.resolve(AbstractFileResolver.java:60)
	at org.gradle.api.internal.file.DefaultFileOperations.file(DefaultFileOperations.java:69)
	at org.gradle.api.internal.project.AbstractProject.file(AbstractProject.java:620)
	at org.gradle.groovy.scripts.DefaultScript.file(DefaultScript.java:113)
	at org.gradle.api.internal.BeanDynamicObject$GroovyObjectAdapter.invokeMethod(BeanDynamicObject.java:268)
	at org.gradle.api.internal.BeanDynamicObject.invokeMethod(BeanDynamicObject.java:129)
	at org.gradle.api.internal.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:78)
	at build_1tk45g2ye7zc7fvkpg0vvppds$_run_closure1_closure3_closure8.doCall(/Users/lindafiscus/github/Easy_xkcd/app/build.gradle:7)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:63)
	at org.gradle.api.internal.AbstractNamedDomainObjectContainer.create(AbstractNamedDomainObjectContainer.java:59)
	at org.gradle.api.internal.AbstractNamedDomainObjectContainer.create(AbstractNamedDomainObjectContainer.java:52)
	at org.gradle.api.internal.NamedDomainObjectContainerConfigureDelegate._configure(NamedDomainObjectContainerConfigureDelegate.java:39)
	at org.gradle.api.internal.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:73)
	at build_1tk45g2ye7zc7fvkpg0vvppds$_run_closure1_closure3.doCall(/Users/lindafiscus/github/Easy_xkcd/app/build.gradle:4)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:63)
	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:130)
	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:91)
	at org.gradle.api.internal.AbstractNamedDomainObjectContainer.configure(AbstractNamedDomainObjectContainer.java:68)
	at org.gradle.api.internal.AbstractNamedDomainObjectContainer.configure(AbstractNamedDomainObjectContainer.java:24)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:55)
	at org.gradle.api.Action$execute.call(Unknown Source)
	at com.android.build.gradle.BaseExtension.signingConfigs(BaseExtension.groovy:318)
	at com.android.build.gradle.AppExtension_Decorated.signingConfigs(Unknown Source)
	at build_1tk45g2ye7zc7fvkpg0vvppds$_run_closure1.doCall(/Users/lindafiscus/github/Easy_xkcd/app/build.gradle:3)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:63)
	at org.gradle.api.internal.plugins.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:145)
	at org.gradle.api.internal.plugins.ExtensionsStorage.configureExtension(ExtensionsStorage.java:69)
	at org.gradle.api.internal.plugins.DefaultConvention$ExtensionsDynamicObject.invokeMethod(DefaultConvention.java:207)
	at org.gradle.api.internal.CompositeDynamicObject.invokeMethod(CompositeDynamicObject.java:147)
	at org.gradle.groovy.scripts.BasicScript.methodMissing(BasicScript.java:79)
	at build_1tk45g2ye7zc7fvkpg0vvppds.run(/Users/lindafiscus/github/Easy_xkcd/app/build.gradle:2)
	at org.gradle.groovy.scripts.internal.DefaultScriptRunnerFactory$ScriptRunnerImpl.run(DefaultScriptRunnerFactory.java:52)
	... 43 more
2015-11-17 15:20:53,971 [58749561]   WARN - nal.AbstractExternalSystemTask - Cannot convert URL 'C:/' to a file. 
com.intellij.openapi.externalSystem.model.LocationAwareExternalSystemException: Cannot convert URL 'C:/' to a file.
	at org.jetbrains.plugins.gradle.service.project.AbstractProjectImportErrorHandler.createUserFriendlyError(AbstractProjectImportErrorHandler.java:103)
	at org.jetbrains.plugins.gradle.service.project.BaseProjectImportErrorHandler.getUserFriendlyError(BaseProjectImportErrorHandler.java:158)
	at org.jetbrains.plugins.gradle.service.project.BaseGradleProjectResolverExtension.getUserFriendlyError(BaseGradleProjectResolverExtension.java:438)
	at com.android.tools.idea.gradle.project.AndroidGradleProjectResolver.getUserFriendlyError(AndroidGradleProjectResolver.java:348)
	at org.jetbrains.plugins.gradle.service.project.AbstractProjectResolverExtension.getUserFriendlyError(AbstractProjectResolverExtension.java:164)
	at org.jetbrains.plugins.gradle.service.project.GradleProjectResolver$ProjectConnectionDataNodeFunction.fun(GradleProjectResolver.java:366)
	at org.jetbrains.plugins.gradle.service.project.GradleProjectResolver$ProjectConnectionDataNodeFunction.fun(GradleProjectResolver.java:332)
	at org.jetbrains.plugins.gradle.service.project.GradleExecutionHelper.execute(GradleExecutionHelper.java:225)
	at org.jetbrains.plugins.gradle.service.project.GradleProjectResolver.resolveProjectInfo(GradleProjectResolver.java:97)
	at org.jetbrains.plugins.gradle.service.project.GradleProjectResolver.resolveProjectInfo(GradleProjectResolver.java:65)
	at com.intellij.openapi.externalSystem.service.remote.RemoteExternalSystemProjectResolverImpl$1.produce(RemoteExternalSystemProjectResolverImpl.java:41)
	at com.intellij.openapi.externalSystem.service.remote.RemoteExternalSystemProjectResolverImpl$1.produce(RemoteExternalSystemProjectResolverImpl.java:37)
	at com.intellij.openapi.externalSystem.service.remote.AbstractRemoteExternalSystemService.execute(AbstractRemoteExternalSystemService.java:59)
	at com.intellij.openapi.externalSystem.service.remote.RemoteExternalSystemProjectResolverImpl.resolveProjectInfo(RemoteExternalSystemProjectResolverImpl.java:37)
	at com.intellij.openapi.externalSystem.service.remote.wrapper.ExternalSystemProjectResolverWrapper.resolveProjectInfo(ExternalSystemProjectResolverWrapper.java:49)
	at com.intellij.openapi.externalSystem.service.internal.ExternalSystemResolveProjectTask.doExecute(ExternalSystemResolveProjectTask.java:51)
	at com.intellij.openapi.externalSystem.service.internal.AbstractExternalSystemTask.execute(AbstractExternalSystemTask.java:138)
	at com.intellij.openapi.externalSystem.service.internal.AbstractExternalSystemTask.execute(AbstractExternalSystemTask.java:124)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$4.execute(ExternalSystemUtil.java:540)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$5$1.run(ExternalSystemUtil.java:612)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
	at com.intellij.openapi.progress.impl.CoreProgressManager$8.run(CoreProgressManager.java:367)
	at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:452)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
	at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:617)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:400)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)

2015-11-17 15:20:54,999 [58750589]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:20:57,469 [58753059]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:20:57,622 [58753212]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:20:59,050 [58754640]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:20:59,185 [58754775]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:20:59,359 [58754949]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
2015-11-17 15:21:50,630 [58806220]   WARN - ij.ui.messages.SheetController - URL is null; HyperlinkEvent: javax.swing.event.HyperlinkEvent[source=javax.swing.JEditorPane[,120,45,311x39,layout=javax.swing.plaf.basic.BasicTextUI$UpdateHandler,alignmentX=0.0,alignmentY=0.0,border=javax.swing.plaf.basic.BasicBorders$MarginBorder@600ccf6f,flags=16777504,maximumSize=,minimumSize=,preferredSize=,caretColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],disabledTextColor=javax.swing.plaf.ColorUIResource[r=187,g=187,b=187],editable=false,margin=javax.swing.plaf.InsetsUIResource[top=0,left=0,bottom=0,right=0],selectedTextColor=javax.swing.plaf.ColorUIResource[r=255,g=255,b=255],selectionColor=javax.swing.plaf.ColorUIResource[r=47,g=101,b=202],kit=javax.swing.text.html.HTMLEditorKit@240906be,typeHandlers={text/html=javax.swing.text.html.HTMLEditorKit@240906be}]] 
 ```

I apologize if this is not an issue with your repository, as I said, I'm new to this.

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -1,14 +1,6 @@
 apply plugin: 'com.android.application'
 android {
-    signingConfigs {
-        config {
-            keyAlias ''
-            keyPassword ''
-            storeFile file('C:/')
-            storePassword ''
-        }
-        release
-    }
+    
     compileSdkVersion 23
     buildToolsVersion "23.0.1"
     defaultConfig {

